script/verification/VerifyClisBnbStrategy.s.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.24;
+
+import {IVault} from "lib/yieldnest-vault/src/interface/IVault.sol";
+import {BaseScript} from "script/BaseScript.sol";
+import {RulesVerification} from "./RulesVerification.sol";
+import {ISlisBnbProvider} from "src/interfaces/ISlisBnbProvider.sol";
+import {IValidator} from "lib/yieldnest-vault/src/interface/IValidator.sol";
+import {SafeRules} from "lib/yieldnest-vault/script/rules/SafeRules.sol";
+import {Test} from "forge-std/Test.sol";
+import {RolesVerification} from "./RolesVerification.sol";
+
+// FOUNDRY_PROFILE=mainnet forge script VerifyClisBnbStrategy
+contract VerifyClisBnbStrategy is BaseScript, Test {
+    function symbol() public pure override returns (string memory) {
+        return "ynClisBnb";
+    }
+
+    function run() public {
+        _loadDeployment();
+        _setup();
+
+        verify();
+    }
+
+    function verify() internal view {
```

Comment:
the most critical thing to verify out of this entire script is to verify upgradeability

Verify ProxyAdmin is owned by a timelock that is controlled by ADMIN and has one day delay.

If that is bricked, we can't upgrade anymore

Reply:
you can use RolesVerification